src/aggregate_score.rs

```diff
@@ -54,6 +71,38 @@ pub fn sample_aggregate_scores<const N_HAND: usize, R: Rng>(rng: &mut R, num_sam
     scores
 }
 
+pub fn parallel_sample_aggregate_scores<const N_HAND: usize, R: Rng>(
+    _rng: &mut R,
+    num_samples: u32,
+    number_of_threads: u32,
```

Comment:
nit: use `num` or `number` for both to be consistent. It looks like `num_` is a bit more common.